tests/test_apel_plugin.py

```diff
@@ -21,3 +25,90 @@ async def test_regex_dict_lookup(self):
             await functions.regex_dict_lookup(term_c, dict)
```

Comment:
The fact that `regex_dict_lookup` can kill the program is rather extreme. Generally you would rather throw an Exception and let the caller decide what to do. (This is afaik also what a dict does if you want to get the value of a non-existing key.) Same applies to logging; I would have the caller log the error, because the caller has more context which could be added to the log message. 

`regex_dict_lookup` is a rather simple function which is by itself completely independent of your program. By making it crash the program if the regex doesn't match, you unnecessarily infiltrate it with context from your program which is not really good design.

I typically only crash the program when there are unrecoverable errors or bugs. Otherwise I'll hand the error to the caller who can decide to either deal with it or pass it to the next caller. Once it hits main and it can't recover, the program is crashed. (Admittedly this is much more painful to do in Python than it is in Rust).

Reply:
This function is currently used for mapping the `user_id ` to VOMS attributes defined in the config file. Once this information is available in the AUDITOR database, `regex_dict_lookup` can be removed.

Reply:
Then I don't think there is much point in testing this, but it also doesn't hurt ;)

Reply:
Done with commit [b349e26](https://github.com/ALU-Schumacher/AUDITOR-APEL-plugin/pull/22/commits/b349e2644bca8e151bf7729dbe82dc9743427c02)